helpers/setURLQueryParam.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const URL = require('url').URL;
+const utils = require('handlebars-utils');
+const common = require('./lib/common.js');
+const SafeString = require('handlebars').SafeString;
+
+const factory = () => {
+    return function(string, key, value) {
+        if (string instanceof SafeString) {
+            string = string.toString();
+        }
+        if (!utils.isString(string) || !common.isValidURL(string)
+            || !utils.isString(key) || !utils.isString(value)
```

Comment:
Same here if key and value exists and if they are not string throw type error.

Reply:
It seems like in most helpers instead of throwing an error we just return some default output (empty string or pass original value through). Why is it better to throw an error here?

Reply:
I am suggesting type error here bcoz if the values do exists and if they not string they are breaking a contract defined for the method which is why I recommended throwing an error so it is explicit to the developer that he is using wrong input.

If we do return some value back it creates confusion as to why they are not receiving what they expected.

Reply:
So they'd be able to see this error in CLI, right? What happens in prod, does the whole page fail?